packages/repository-tests/src/crud/replace-by-id.suite.ts

```diff
@@ -34,6 +34,11 @@ export function createSuiteForReplaceById(
     })
     id: MixedIdType;
 
+    // cloudant needs this property to do replacement method
+    // see cloundant README file for more details
```

Comment:
[Cloudant README](https://github.com/strongloop/loopback-connector-cloudant#_rev-property)

Reply:
I find it suboptimal to force all connector tests (SQL, MongoDB, etc.) to use `_rev` property. Let's find a way how to allow individual connectors to provide any extra properties via `CrudFeatures`.

Reply:
I see that you added `revisionTokenRequired` flag to fix test failures.

Personally, I'd prefer to find a way how to define `_rev` property only when running on Cloudant, so that we are not using `_rev` property when testing e.g. MongoDB.

Having said that, I don't have any solution I could offer from the top of my head, so it's probably better to accept the current proposal for now and get this pull request landed sooner rather than later 👍